app/import.go

```diff
@@ -1097,7 +1097,7 @@ func (a *App) ImportUserTeams(user *model.User, data *[]UserTeamImportData) *mod
 			a.UpdateTeamMemberSchemeRoles(team.Id, user.Id, isSchemeUser, isSchemeAdmin)
 		}
 
-		if defaultChannel, err := a.GetChannelByName(model.DEFAULT_CHANNEL, team.Id); err != nil {
+		if defaultChannel, err := a.GetChannelByName(model.DEFAULT_CHANNEL, team.Id, false); err != nil {
```

Comment:
For now, this is safe here, but... makes sense to allow override archived channels, or actually, make sense to import Archived channels (Adding the DeleteAt field to the import)?

Reply:
Good question. Probably something to ask Jason (and can be another ticket if he thinks we should support it).

Reply:
The most forgiving thing is to allow users to import archived channels. If we didn't think it were valuable then we wouldn't be providing access with this feature in the first place.

Reply:
Probably make sense if we want to allow customers to import from slack through the importer.